automated install/basic-install.sh

```diff
@@ -90,7 +90,7 @@ backupLegacyPihole(){
 		echo "Original Pi-hole detected.  Initiating sub space transport"
 		$SUDO mkdir -p /etc/pihole/original/
 		$SUDO mv /etc/dnsmasq.d/adList.conf /etc/pihole/original/adList.conf.$(date "+%Y-%m-%d")
-		$SUDO mv /etc/dnsmasq.conf /etc/pihole/original/dnsmasq.conf.$(date "+%Y-%m-%d")
+		$SUDO mv /etc/dnsmasq.d/01-pihole.conf /etc/pihole/original/dnsmasq.d/01-pihole.conf.$(date "+%Y-%m-%d")
```

Comment:
This doesn't need to be modified. The backupLegacyPihole() only was to check for Pi-Hole v1, and won't be triggered unless someone is running very old code. This function is set for depreciation soon anyways.